lib/index.js

```diff
@@ -6,9 +6,9 @@ import FS from 'fs';
 import { dirname } from 'path';
 import YAML from 'js-yaml';
 
-const makeEngineString = engineNames => (
+const getEngineArgs = engineNames => (
   engineNames.map(language => (`-e ${language}`))
-).join(' ');
+).join(' ').split(' ').filter(i => i);
```

Comment:
Suppose `engineNames = ['engine1', 'engine2']`, then:

1. `map(...)` produces `['-e engine1', '-e engine2']`.
1. `join(...)` produces `'-e engine1 -e engine2'`.
1. `split(...)` produces `['-e', 'engine1', '-e', 'engine2']`.
1. `flter(...)` removes empty elements (if `engineNames = []`, then steps 1-3 produces `['']`, and this transforms it to `[]`).

If you remove `.join(' ').split(' ')` then you get result from step 1, except for the removal of the empty string from the resulting array (step 4), but executing directly codeclimate requires the args as in step 3.